client/src/app/site/services/operator.service.ts

```diff
@@ -436,7 +442,7 @@ export class OperatorService {
                 this._permissions !== undefined;
         }
         if (this.isAuthenticated) {
-            isReady = isReady && this._OML !== undefined && this._CML !== undefined;
+            isReady = isReady && this._OML !== undefined && this._CML !== undefined && this._userSubject !== undefined;
```

Comment:
this._userSubject is never going to be undefined, this._userSubject.value may be.